### ISSUE TITLE
Could not manage to make JSON work

### ISSUE THREAD

**Issue by gmetais** [claudetech/grunt-i18n-static#3]
Hi and thank you this Grunt plugin!

I didn't manage to make it work with the JSON format. It looks like the json files are not read at all.
It worked like a charm as soon as I switched to YML.

My grunt conf:

```
    i18n: {
      dist: {
        options: {
          baseDir: 'src',
          outputDir: 'translated',
        }
      },
      options: {
        fileFormat: 'json',
        locales: ['en', 'fr'],
        locale: 'en',
        localesPath: 'src/locales'
      }
    }
```

My project structure and my JSON files were good.

From what I see in the node-static-i18n code, the JSON files are never read. Have a look at the `return` here: https://github.com/claudetech/node-static-i18n/blob/master/src/index.coffee#L52

Do you think it's a bug, or did I make something wrong?

**Comment by danhper** [comment]
Hi,
I think that i18next is supposed to load JSON files automatically, and that [seems to be working](https://github.com/claudetech/node-static-i18n/blob/master/test/index-test.coffee#L26)
but as I usually use yaml, something could have been broken by some change in i18next or something and I did not notice.
I am going to check this out!

**Comment by gmetais** [comment]
Thank you!

**Comment by danhper** [comment]
It was a bug when using JSON + custom locale path, it should be fixed in 0.1.11.
Thanks for reporting!

The path for i18next to load JSON [was not set properly](https://github.com/claudetech/node-static-i18n/blob/master/src/index.coffee#L60) as `i18n.resGetPath` got a default value in the grunt plugin.

**Comment by gmetais** [comment]
Great thanx!
